src/SFA.DAS.RoatpOversight.Web.UnitTests/Controllers/Oversight/OversightOutcomeValidatorTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Linq;
+using NUnit.Framework;
+using SFA.DAS.RoatpOversight.Web.Validators;
+
+namespace SFA.DAS.RoatpOversight.Web.UnitTests.Controllers.Oversight
+{
+    [TestFixture]
+    public class OversightOutcomeValidatorTests
+    {
+        [TestCase("", true)]
+        [TestCase(null, true)]
+        [TestCase("successful", false)]
```

Comment:
Should these be upper case, and const values?

Reply:
I've used OversightReviewStatus consts in the APR-1572 branches

Reply:
I'll pull that in and use it, thanks